tests/core/test_channels.py

```diff
@@ -750,6 +880,66 @@ async def test_slackbot_send_attachment_only():
         }
 
 
+@pytest.mark.filterwarnings("ignore:unclosed.*:ResourceWarning")
+@pytest.mark.asyncio
+async def test_slackbot_send_attachment_only_threaded():
+    from rasa.core.channels.slack import SlackBot
+
+    with aioresponses() as mocked:
+        mocked.post(
+            "https://www.slack.com/api/chat.postMessage",
+            payload={"ok": True, "purpose": "Testing bots"},
+        )
+
+        bot = SlackBot("DummyToken", "General", ts="DummyThread")
+        attachment = {
```

Comment:
This attachment is exactly the same as the one used in `test_slackbot_send_attachment_only`.
Might make sense to define it outside of the tests as a constant in the module, to avoid code duplication and make potential future addition of keys easier.